package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/search-ui-react",
-  "version": "1.0.0-beta.269",
+  "version": "1.0.0-beta.298",
```

Comment:
if QA is happening for all of the libraries, do we want to release beta versions of Core and Headless first?

Reply:
published beta version for core, headless, and headless-react.